docs/stories/docker/mailhog-service.md

```diff
@@ -0,0 +1,14 @@
+# Mailhog service
+For development environments, the service mailhog helps to test emails locally, it's declared in the [docker-compose](../../../docker-compose.yml) and exports two ports, the first one is 1025 that it's used for smtp server, and the 8025 is a web interface, so you can see all test emails.
+
+For Inter-stack containers communication, this service can be used with its name and port, for example:
```

Comment:
I think it's possible to configure Magento via environment variables in this case

```
https://devdocs.magento.com/guides/v2.4/config-guide/deployment/pipeline/example/environment-variables.html#convert-configuration-paths-to-variable-names
```

Docs here: https://docs.magento.com/user-guide/system/email-communications.html

Reply:
Perfect